plugins/catalog-backend-module-github/src/lib/github.ts

```diff
@@ -52,6 +52,19 @@ export type UserResponse = {
   organizations?: Connection<GithubOrg>;
 };
 
+/**
+ * Query Options
+ *
+ * @public
+ *
+ * pageSize - The number of items to fetch per page
+ * requestDelayMs - The delay between requests in milliseconds
+ */
+export type QueryOptions = {
+  pageSize?: number;
+  requestDelayMs?: number;
```

Comment:
```suggestion
 * requestDelay - The delay between requests
 */
export type QueryOptions = {
  pageSize?: number;
  requestDelay?: string | HumanDuration;
```

as used in other places throughout the project. Then read it [like this](https://github.com/backstage/backstage/blob/1a03dc6f69341df1f4b20a53b60d8f09cfaa2e05/plugins/notifications-backend-module-email/src/processor/NotificationsEmailProcessor.ts#L88). Now users can say `requestDelay: 3 seconds` or `500ms` or `{ seconds: 1 }` so it's clear what unit is used.